src/Omega_h_vector.hpp

```diff
@@ -150,6 +150,16 @@ OMEGA_H_INLINE Vector<n> zero_vector() OMEGA_H_NOEXCEPT {
   return fill_vector<n>(0.0);
 }
 
+template <Int n>
+OMEGA_H_INLINE Vector<n> max_float_vector() OMEGA_H_NOEXCEPT {
+  return fill_vector<n>(Kokkos::Experimental::finite_max_v<float>);
```

Comment:
Please replace `float` with `Real` as that is the Omega_h type for floating point values.  This also applies to `min_float_vector`.